resources/js/pages/invoices/index.tsx

```
@@ -0,0 +1,140 @@
+    const [deleting, setDeleting] = useState<string | null>(null);
+
+    const formatCurrency = (amount: number) => {
+        return new Intl.NumberFormat('id-ID', {
+            style: 'currency',
+            currency: 'IDR',
+            minimumFractionDigits: 0,
+        }).format(amount);
+    };
+
+    const handleDelete = () => {
+        if (!deleting) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)